bundles/org.openhab.binding.nuvo/README.md

```diff
@@ -47,8 +47,8 @@ The thing has the following configuration parameters:
 
 Some notes:
 
-* The direct connection to the MPS4 server has not been exhaustively tested, please report any issues found.
-* The only issue with the MPS4 connection seen thus far is that the setting SxDISPINFO as seen in the advanced rules below does not work.
+* If the port is set to 5006 the binding will adjust its protocol to connect to a NuVo via an MPS4 IP connection.
+* MPS4 connections ignore the SxDISPINFO command as seen in the advanced rules below does not work.
```

Comment:
This is a little awkward,  how about this:

> * MPS4 connections do not support SxDISPINFO commands including those outlined in the advanced rules section below.

Reply:
done